### PR TITLE
HOCS-1751

### PR DESCRIPTION
**Pull request by Paulcr-ho**
Removed the kube namespace and replaced with environment.
The drone command will need to pass the new environment (like it passed the namespace) and will omit the namespace.

Note: There is an unknown in the working of the semver environment variable which may break the build.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -48,10 +48,10 @@ pipeline:
       - docker_password
       - git_password
     commands:
-      - /app/hocs-deploy --version=$${SEMVER} --serviceGitToken=$${GIT_PASSWORD} --service=hocs-templates --gitToken=$${GITHUB_PASSWORD} --gitRepo="https://gitlab.digital.homeoffice.gov.uk/hocs/hocs-versions.git" --environment=qa --dockerRepository=quay.io/ukhomeofficedigital --sourceBuild=$${IMAGE_VERSION} --registryUser=ukhomeofficedigital+hocs --registryPassword=$${DOCKER_PASSWORD}
+      - /app/hocs-deploy --version=$${SEMVER} --serviceGitToken=$${GIT_PASSWORD} --service=hocs-templates --gitToken=$${GITHUB_PASSWORD} --gitRepo="https://gitlab.digital.homeoffice.gov.uk/hocs/hocs-versions.git" --environment=$${ENVIRONMENT} --dockerRepository=quay.io/ukhomeofficedigital --sourceBuild=$${IMAGE_VERSION} --registryUser=ukhomeofficedigital+hocs --registryPassword=$${DOCKER_PASSWORD}
```
Comment (zuzak) [inline .drone.yml:51]: This might not work as `$ENVIRONMENT` isn't specified?

Comment (Paulcr-ho) [reply .drone.yml:51]: This is what I was concerned about in the last line of the description above.
Based on the latest changes in the kube project for templates, the namespace needs to be put in here, so I will modify.

Comment (zuzak) [reply .drone.yml:51]: You could use `$DRONE_DEPLOY_TO`, which is filled with the triggering environment from `when: environment:` (ultimately from the last argument in the `drone deploy` command). You can't use `$KUBE_NAMESPACE` here without setting it somewhere, as it's not exposed to this particular build step.

Comment (Paulcr-ho) [reply .drone.yml:51]: Another option is to use DRONE_DEPLOY_TO

Comment (Paulcr-ho) [reply .drone.yml:51]: Implementing DRONE_DEPLOY_TO

**Pull request merged**